sys/auto_init/auto_init_ng_netif.c

```diff
@@ -0,0 +1,99 @@
+#include "board.h"
+#include "net/ng_nomac.h"
+#include "net/ng_netbase.h"
+
+#define ENABLE_DEBUG 1
+#include "debug.h"
+
+/**
+ * @brief   Define stack parameters for the MAC layer thread
+ * @{
+ */
+#define MAC_STACKSIZE           (KERNEL_CONF_STACKSIZE_DEFAULT)
+#define MAC_PRIO                (PRIORITY_MAIN - 3)
```

Comment:
> The priority is outside of the MAC layer since a user might want to have different priorities for different MAC threads of the same MAC implementation

Can you give a use case for this?